platform/macos/src/MGLMapView.h

```diff
@@ -48,6 +48,7 @@ NS_ASSUME_NONNULL_BEGIN
     ensuring that your use adheres to the relevant terms of use.
  */
 IB_DESIGNABLE
+MGL_EXTERN
```

Comment:
Open MapDocument.xib. Xcode will automatically start building. After awhile, this:

<img width="719" alt="blank" src="https://cloud.githubusercontent.com/assets/1231218/21027216/ae6a532c-bd44-11e6-9a8e-958f3ec57ea7.png">

will turn into this:

<img width="732" alt="designable" src="https://cloud.githubusercontent.com/assets/1231218/21027252/d03fb8f2-bd44-11e6-8251-9ad6fd91881d.png">

It’s a bit underwhelming on macOS, but the iOS SDK version of this displays some important setup instructions, so we should make sure designables in general still work.